test/cypress/integration/a11y.spec.js

```diff
@@ -38,6 +38,14 @@ describe('a11y', () => {
 
     beforeEach(() => {
       tour = setupTour(Shepherd);
+
+      // This is a hack removing the extra page elements so the page does not scroll. Cypress hates scrolling for some reason.
```

Comment:
@chuckcarpenter this hack sucks, but Cypress was refusing to scroll right, and since the scrolling was impossible to get to work, the poppers were always hidden, thus breaking the tests.

Reply:
reasonable, but I guess we'll need to figure out a way to test in scrolling at some point.